code/src/index.css

```diff
@@ -1,13 +1,129 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Roboto', sans-serif;
+  background-image: url("/background-img.jpg");
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h1 {
+ font-family: 'Pacifico', cursive;
 }
+
+h2 {
+  font-size: 1em;
+  font-weight: 600;
+  padding:0;
+}
+
+textarea:focus {
+  outline: none;
+  border: 2px solid pink;
+}
+
+.all-thoughts {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.thought {
+ border: 1px solid black;
+ background-color: white; 
+ box-shadow: 7px 7px black;
+ margin: 25px 0 25px 0;
+ width: 360px;
+ padding: 20px 0 20px 10px;
+}
+
+.messages {
+  max-width: 350px;
+  overflow-wrap: break-word;
+}
+
+.post-thought-container {
+  display: flex;
+  background-color: rgb(224, 224, 224);
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 360px;
+  border: 1px solid black;
+  box-shadow: 7px 7px black;
+}
+
+.post-thought-form textarea{
+  width: 280px;
+  height: 80px;
+  margin-left: 35px;
+}
+
+.send-btn {
+  display: inline-block;
+  border: none;
+  -webkit-appearance: none;
+  margin: 10px 0 10px 20px;
+  padding: 5px 15px 5px 15px;
+  border-radius: 20px;
+}
+
+.send-btn:hover {
+  border: 2px solid pink;
```

Comment:
Perhaps this border could be removed, or add a border with the same appearance, but grey, to .send-btn so that when you hover, it wont "jump".

Reply:
Thank you so much!! I removed it and now it stopped jumping. 😀